umh-core/pkg/fsmv2/examples/cpuhealth_scenario.go

```
@@ -0,0 +1,151 @@
+//	go run pkg/fsmv2/cmd/runner/main.go --scenario cpuhealth
+//
+// What it drives: the fsmv2 CPU monitor worker's Poll over a mocked filesystem,
+// through a healthy (quiet present cgroup judges healthy with capable/measured
+// counts) and an unhappy (unreadable cpu.stat reports could-not-measure) path.
+var CPUHealthScenarioEntry = Scenario{
+	Name:        "cpuhealth",
+	Description: "Drives the CPU monitor worker over a mocked filesystem (healthy + failing read)",
+	YAMLConfig:  "", // worker built directly with a mock-backed sampler
+	CustomRunner: func(ctx context.Context, _ RunConfig) (*RunResult, error) {
+		runCPUHealthScenario(ctx)
+		done := make(chan struct{})
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)